PCLumiProd/writePCLumiCSVs.py

```diff
@@ -405,6 +406,8 @@ def findRunInFill(run):
         if histName.find("Ratio_Correction_")!=-1: 
```

Comment:
Hi Chris and Sam, @capalmer85 @samhiggie,  this looks good to me. However, I'm still worried that some histograms might be empty. When you find the histogram from the correction file, you might need to check if it's empty (i.e. GetBinContent()  is zero)